src/components/Landing.js

```diff
@@ -308,9 +312,8 @@ export default class Landing extends Component {
         let dataPoint = {};
         let startDate = extractDateFromGMTDateString(currentMedicationItem[startDateFieldName]);
         let endDate = extractDateFromGMTDateString(currentMedicationItem[endDateFieldName]);
-        let [oStartDate, oEndDate] = [new Date(startDate), new Date(endDate)];
-        let diffTime = oEndDate - oStartDate;
-        let diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+        let oStartDate = new Date(startDate);
+        let diffDays = getDiffDays(startDate, endDate);
```

Comment:
I can't remember if there are serious defects to using Date objects, but you might want to store datetimes as Date objects (rather than strings), so you can do date math directly (instead of needing to convert to a Date first).

Reply:
@ivan-c I am keeping these as strings here because they will be converted to Date object when the graph plots them. If I store them as date objects now, there would be runtime error, i.e. cannot convert date object to date object.